### PR TITLE
Fix styling of the about page header

### PR DESCRIPTION
**Pull request by pietroalbini**

### CODE REVIEW

File: templates/style/base.scss [hunk 1]
```diff
@@ -447,6 +447,10 @@ div.cratesfyi-package-container {
     h1 {
         margin: 0;
         padding: 15px 0 0 14px;
+
+        &.no-description {
```
Comment (GuillaumeGomez) [inline templates/style/base.scss:451]: You don't need the `&` here.

**Comment by jyn514** [comment]
I kind of liked the underscore before, is there a reason you removed it?
image

**Comment by pietroalbini** [comment]
It's not consistent with the other pages that have tabs.

File: templates/style/base.scss (hunk 1 above)
Comment (pietroalbini) [reply templates/style/base.scss:451]: Why not? I need to apply it to `<h1 class="no-description">`.

Comment (GuillaumeGomez) [reply templates/style/base.scss:451]: Oooooh. I see what you did there! Indeed, my bad!

**Comment by jyn514** [comment]
Seems fine to me.

**Pull request merged**